packages/strapi-generate-plugin/files/admin/src/containers/Initializer/index.js

```diff
@@ -0,0 +1,27 @@
+/**
+ *
+ * Initializer
+ *
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import pluginId from '../../pluginId';
+
+class Initializer extends React.PureComponent {
```

Comment:
Can't be a stateless component with useEffect equivalent instead of componentDidMount ?

Reply:
Well I think we can do something like this:

```
import { useEffect } from 'react';

const Initializer = ({ updatePlugin }) => {
  useEffect(() => {
    updatePlugin(pluginId, 'isReady', true);
  }, [updatePlugin]);

  return null;
};
```

But since, we still have issues with our dependencies especially React I think that switching to the new hook API is a task that needs to be undertaken in a separated PR, once we have a strong end to end test using cypress setup.